### PR TITLE
Updated org people title to use postion rdf label

### LINKED ISSUE #173: Organization individual view should list people positions associated with the organization
All people listings under organization display view use hrJobTitle which does not always reflect the position within the organization.

https://scholars.library.tamu.edu/vivo/display/nb7f16f5c/Organizations/View%20All?People.page=1&People.size=100

image

image

### LINKED ISSUE #189: 3.	Faculty have different titles in different organizations but currently we use their preferred title only for all organization pages. (Priority: medium)
Preference "organization" job title but fall back on HR job title., Filter out all others.
This logic would take the form of a handlebars filter.
Start here: https://github.com/vivo-community/scholars-embed-utilities/blob/main/src/utilities/template.utility.ts#L185

### PR DESCRIPTION
**Pull request by jeremythuff**
Co-authored-by: William Welling <wwelling@library.tamu.edu>

### CODE REVIEW

**Review by rmathew1011 — CHANGES_REQUESTED** [review]
Extra line needed:
src/main/resources/templates/sparql/organization/peopleTitle.sparql

**Comment by ghost** [comment]
@rmathew1011 we are not yet follow posix standard and requiring the newline for these files. If we do, we will have to update all the others for consistency.

File: src/main/resources/templates/sparql/organization/peopleTitle.sparql [hunk 1]
```diff
@@ -20,7 +21,7 @@ CONSTRUCT {
         ?position a vivo:Position .
         ?position vivo:relatedBy ?person .
         ?person a foaf:Person .
-        ?person vivo:hrJobTitle ?title .
-        BIND( CONCAT( STR(?title), REPLACE(STR(?person), "(^.*/)", "::") ) AS ?titleWithId ) .
+        ?position rdfs:label ?label .
+        BIND( CONCAT( STR(?label), REPLACE(STR(?person), "(^.*/)", "::") ) AS ?labelWithId ) .
     }
 }
```
Comment (kaladay) [inline src/main/resources/templates/sparql/organization/peopleTitle.sparql:27]: Traling new line not present.

**Review by kaladay — APPROVED** [review]

File: src/main/resources/templates/sparql/organization/peopleTitle.sparql (hunk 1 above)
Comment (ghost) [reply src/main/resources/templates/sparql/organization/peopleTitle.sparql:27]: Yep, I will get them soon enough. There are a lot of them without and will do them all at once.

Comment (kaladay) [reply src/main/resources/templates/sparql/organization/peopleTitle.sparql:27]: I also just noticed that there are comments on the PR itself about the newline.
I reviewed the code but I didn't think to look at the comments that are not in code review part of the PR.

**Pull request merged**